test_config_templates.py

```diff
@@ -18,29 +17,19 @@
 )
 
 
-@pytest.fixture(scope="module")
-def project(request) -> Project:
-    proj = create_project()
-    yield proj
-
-    def delete_project():
-        proj.delete()
-
-    request.addfinalizer(delete_project)
-
-
 _configs = (Path(__file__).parent / "config_templates").glob("**/*.yml")
 
 
-@pytest.mark.parametrize("config_file", _configs)
-def test_configs(config_file, project: Project):
-    _config_file = str(config_file)
-    _config_dict = yaml.safe_load(open(_config_file).read())
-
-    resp = requests.post(
-        f"https://api-dev.gretel.cloud/projects/{project.name}/models",
-        json=_config_dict,
-        params={"dry_run": "yes"},
-        headers={"Authorization": _api_key}
-    )
-    assert resp.status_code == 200
+@pytest.mark.parametrize(
+    "_config_file", ["/".join(str(_config).split("/")[-4:]) for _config in _configs]
+)
+def test_configs(_config_file):
+    with tmp_project() as project:
```

Comment:
suggestion: could just use a single project for every config, since I think this will create a new project for each config. this could be _maybe_ done with a fixture like...

```python
@pytest.fixture(scope="session")
def tmp_project() -> Project:
    with tmp_project() as project:
        yield project
```